hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```diff
@@ -440,8 +440,10 @@ private XceiverClientReply sendCommandWithRetry(
       return reply;
     } else {
       Objects.requireNonNull(ioException);
-      LOG.error("Failed to execute command {} on the pipeline {}.",
-          processForDebug(request), pipeline);
```

Comment:
Why don't we follow the similar pattern? Single liner log when non debug and full verbose with debug?